### PR TITLE
Fixed RobotJS name.

### PR DESCRIPTION
**Pull request by octalmage**
There's no space!

### CODE REVIEW

**Comment by dthree** [comment]
:)

File: readme.md [hunk 1]
```diff
@@ -161,4 +161,4 @@ Whichever your preference, Dash and Wat intend to work together to cover all bas
 
 ## License
 
-MIT
\ No newline at end of file
+MIT
```
Comment (Qix-) [inline readme.md]: Keep changes relevant to the commit in the future.

**Pull request merged**